spec/GraphQLOverHTTP.md

```diff
@@ -568,6 +557,11 @@ of `2xx` or `5xx` status codes when responding to invalid requests using the
 Note: URLs that enable GraphQL requests may enable other types of requests - see
 the [URL](#url) section.
 
+Note: When the response media type is `application/json`, clients may use a
+`2xx` status code as an indication that the body contains a well-formed _GraphQL
+response_. See [processing a response](#sec-Processing-a-response) for more
+details.
```

Comment:
This seems to essentially mirror the first paragraph of this section.  Perhaps a more useful non-normative note would be a negative example, such as:

```suggestion
Note: When the response media type is `application/json`, clients cannot count on
the response being a well-formed _GraphQL response_ unless the `2xx` status code
is also returned. See [processing a response](#sec-Processing-a-response) for more
details.
```

Reply:
Or:

```suggestion
Note: When the response media type is `application/json` and the status code is 4xx,
clients cannot count on the response being a well-formed _GraphQL response_.
See [processing a response](#sec-Processing-a-response) for more details.
```

Reply:
I went with your last solution with a [few tweaks](https://github.com/graphql/graphql-over-http/pull/304/commits/f73fd2c5a59714ff5a1b4247496b019e13b2f104)

FWIW, what I'm trying to do in this note is make the parsing of the section easier from a client point of view because the real information is in section 6.1 (body) but it's easy to come to section 6.4 to know how to interpret the status code.

Reply:
Rereading the re-rendered document, I think there is some repetition indeed so I backtracked and just [added the "processing a response" link to the first note](https://github.com/graphql/graphql-over-http/pull/304/commits/110d6adab7b832d3d2ba88fe72ac8279a636907e)